features/account_creation.feature

```diff
@@ -17,18 +17,18 @@ Feature: User Account Creation
     Given I do not have an account on the site
     When I visit the site root path
     Then I am presented with a login page
-    When I click "Sign up"
+    When I click the "Sign up" link
     And I enter my email address
     And I enter a password with incorrect confirmation
-    And I click "submit"
+    And I click the "Sign up" button
     Then I am notified that my password confirmation does not match
 
   Scenario: Invalid Email Format
     Given I do not have an account on the site
     When I visit the site root path
     Then I am presented with a login page
-    When I click "Sign up"
+    When I click the "Sign up" link
     And I enter "garbage" as my email address
     And I enter a password with correct password confirmation
-    And I click "submit"
+    And I click the "Sign up" button
     Then I am notified that my email address is invalid.
```

Comment:
I think all of these changes are good, and improve the cucumber feature file. In fact, I think this should be made as a PR to the curriculum repo. Would you mind doing that for me?